@uportal/waffle-menu/vue.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  filenameHashing: false,
+  publicPath: './',
+  productionSourceMap: false,
+  configureWebpack: {
```

Comment:
These options appear to match the defaults from vue cli.
could the be omitted?

Reply:
This is an optional file.  Is it recommended we don't use them?

Reply:
If we're not using it, then we shouldn't use it :slightly_smiling_face:

Reply:
Ok, I'll remove it since that doesn't appear to be the standard for uPortal components :)